src/Dialect/ONNX/ONNXOps.td.inc

```diff
@@ -82,7 +82,7 @@ def ONNXAcoshOp:ONNX_Op<"Acosh",
 }
 
 def ONNXAddOp:ONNX_Op<"Add",
-  [NoSideEffect, DeclareOpInterfaceMethods<ShapeInferenceOpInterface>]> {
+  [NoSideEffect, DeclareOpInterfaceMethods<ShapeInferenceOpInterface>, DeclareOpInterfaceMethods<FoldOpInterface>]> {
```

Comment:
Instead of modifying this file directly please make the changes in gen_onnx_mlir.py.

Reply:
Fixed